pulp_rpm/tests/functional/api/test_consume_content.py

```diff
@@ -36,9 +35,8 @@ def test_all(self):
         `Pulp #3204 <https://pulp.plan.io/issues/3204>`_
         """
         cfg = config.get_config()
-        try:
-            cli.PackageManager._get_package_manager(cfg)  # pylint:disable=protected-access
-        except NoKnownPackageManagerError:
+        pkg_mgr = cli.PackageManager(cfg)
+        if pkg_mgr.name not in ('dnf', 'yum'):
```

Comment:
@kersommoura 
what do you think about adding a new method?

```python
if not pkg_mgr.is_available():   # this method checks the Exception and returns boolean
    ...
```

That would be a good option, otherwise we should keep catching the exception here.

Reply:
I updated catching the exception here. But we can evaluate another solution as well.